src/Indicator.vala

```diff
@@ -330,6 +335,49 @@ public class Session.Indicator : Wingpanel.Indicator {
         current_dialog.set_transient_for (indicator_icon.get_toplevel () as Gtk.Window);
         current_dialog.show_all ();
     }
+
+    private async void update_tooltip () {
+        string description;
+
+        if (server_type == Wingpanel.IndicatorManager.ServerType.SESSION) {
+
+            if (active_user == null) {
+                debug ("active_user is null. Getting active user");
+                active_user = yield manager.get_active_user ();
+            }
+
+            string active_real_name;
+            int n_online_users = (yield manager.get_n_active_and_online_users ()) - 1;
+
+            // If active user is still null, fallback to "Unknown"
+            if (active_user == null) {
+                critical ("Active user null. Cannot get real name");
+                active_real_name = _("Unknown");
+            } else {
+                active_real_name = active_user.get_real_name ();
+            }
+
+            if (n_online_users > 0) {
+                description = ngettext (
+                    _("Logged in as %s, %i other user logged in"),
+                    _("Logged in as %s, %i other users logged in"),
```

Comment:
```suggestion
                    "Logged in as %s, %i other user logged in",
                    "Logged in as %s, %i other users logged in",
```

Reply:
Won't this stop the tooltip being translated?

Reply:
Nope, this is correct because we're using ngettext here.

Normal gettext: `_("this is a translatable text")`
ngettext: `ngettext("this is a translatable text", "these are translatable texts", text_count)`

So this should work as expected.